### PR TITLE
docs: add some warnings regarding the nullable type variable for enti…

### PR DESCRIPTION
**Pull request by TheoSrz69**
…ty properties

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -71,6 +71,26 @@ The `UploadableField` annotation has a few options. They are as follows:
 * `originalName`: the property that will contain the original name of the uploaded file.
 * `dimensions`: the property that will contain the dimensions of uploaded **image file**
 
+_Warning__ : all these entity properties, in their respective getter & setter methods,  should accept null value.
+It is mandatory for removing correctly entity with symfony. For that, simply add an "?" symbol before the return/send variable type.
```
Comment (garak) [inline docs/usage.md:75]: The documentation is already showing the proper "?" symbols, so I guess we should suggest, instead, to ensure that such symbols are present (if the user followed the examples, they should have already added them).
Moreover, "Symfony" should be capitalized.

**Review by garak — CHANGES_REQUESTED** [review]

File: docs/usage.md (hunk 1 above)
Comment (TheoSrz69) [reply docs/usage.md]: You're (partially) right. The example is good, but it's only an example. Lot of people would not pay attention to detail if document doesn't mention it. I reduced the explanation for being more brief.

Comment (garak) [reply docs/usage.md]: Still a couple of issues: I see a double underscore after "Warning", I guess it's a typo; and we should highlight the newly added paragraph (see create_a_custom_directory_namer.md for an example)

Comment (garak) [reply docs/usage.md]: bump

**Comment by garak** [comment]
Closing.
Feel free to open a new PR if relevant

**Pull request closed without merging**